src/test/shell/bazel/external_integration_test.sh

```diff
@@ -880,6 +880,14 @@ EOF
 }
 
 function test_sha256_weird() {
+  LOG='The SHA256 checksum specified in the WORKSPACE file does not match the SHA256
+checksum of the downloaded file. The checksum exists to ensure a secure build,
+since remote files can change. If the file is correct, update the SHA256 checksum.
+("sha256 = ") in your WORKSPACE file. Otherwise, this could be a security compromise.
+ URL:
+http://127.0.0.1:$fileserver_port/repo.zip
```

Comment:
Here you probably meant `$fileserver_port` to be interpolated, but that does not happen in single quotes; so you have to terminate the single quote and start it after the interpolation.